src/api/resource-service-api/concepts.ts

```diff
@@ -0,0 +1,30 @@
+import { resourceServiceApiGet } from './host';
+import { Concept } from '../../types';
+
+const mapSorting = ({ sortfield }: any) =>
+  sortfield === 'harvest.firstHarvested'
+    ? { field: 'harvest.firstHarvested', direction: 'desc' }
+    : undefined;
+
+export const searchConcepts = (body: any) =>
+  resourceServiceApiGet('/concepts', body?.filters[0]);
+
+export const extractConcepts = (searchResponse: any) =>
+  searchResponse?.hits ?? [];
+
+export const extractConceptAggregations = (searchResponse: any) =>
+  searchResponse.aggregations ?? [];
+
+export const extractConceptsTotal = (searchResponse: any) =>
+  searchResponse?.page?.totalElements ?? 0;
+
+export const paramsToSearchBody = ({ q, page, size, ...params }: any) => ({
+  q,
+  page: page ? Number(page) : undefined,
+  size,
+  sorting: mapSorting(params),
+  filters: []
+});
```

Comment:
Dunno om denne er nødvendig, men POST håndterer feltet `ids`, slik at du kan filtrere svaret på en liste med id'er
```suggestion
export const paramsToSearchBody = ({ ids }: any) => ({
  ids
});
```